src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -900,6 +915,7 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
         // Register translation resources
         if ($dirs) {
             foreach ($dirs as $dir) {
+                $container->addResource(new DirectoryExistenceResource($dir));
```

Comment:
this is wrong. It must be done earlier, because the directories are not added in ``$dirs`` if they don't exist, and this is precisely the case where we need the DirectoryExistenceResource